CovertInfiltration/Src/CovertInfiltration/Classes/X2EventListener_Infiltration_UI.uc

```diff
@@ -7,6 +7,8 @@
 
 class X2EventListener_Infiltration_UI extends X2EventListener config(UI);
 
+var config int MaxOverLimitTillRed;
```

Comment:
Please rename to `MaxOverBarracksLimitTillRed`. Right now it's not clear what behaviour this variable controls